packages/shared/components/popups/CrashReporting.svelte

```diff
@@ -0,0 +1,34 @@
+<script lang="typescript">
+    import { onDestroy } from 'svelte'
+    import { Button, Checkbox, Text } from 'shared/components'
+    import { appSettings, isAwareOfCrashReporting } from 'shared/lib/appSettings'
+    import { localize } from 'shared/lib/i18n'
+    import { closePopup as closePopupInternal } from 'shared/lib/popup'
+    import { AppSettings } from 'shared/lib/typings/app'
+
+    let sendCrashReports = true
+
+    const closePopup = () => {
+        isAwareOfCrashReporting.set(true)
+
+        closePopupInternal()
+    }
+
+    const handleConfirmClick = () => {
+        appSettings.set(<AppSettings>{ ...$appSettings, sendCrashReports, })
```

Comment:
If the user checks the checkbox, I think we need to tell the user to restart Firefly (or use [`app.relaunch()`](https://www.electronjs.org/docs/v14-x-y/api/app#apprelaunchoptions), otherwise Sentry won't be initialized for the current session

Reply:
Yes, this was intentional. Maybe we could simply say something like "This will take effect the next time you start Firefly" rather then telling them to restart? If not that and we add the text hint here for "Please restart Firefly...", then we should also add it back to the settings.

Reply:
Yeah, I think we might want to add it back to the settings in this case